src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -1851,7 +1845,7 @@ static object Test(bool x)
                 context.CompileAssignment("x", "Test(x is string i)", out error, testData);
                 testData.GetMethodData("<>x.<>m0<T>").VerifyIL(
     @"{
-  // Code size       63 (0x3f)
+  // Code size       74 (0x4a)
```

Comment:
>// Code size       74 (0x4a) [](start = 2, length = 28)

It looks like IL got worse

Reply:
Already recorded in [https://github.com/dotnet/roslyn/issues/23222](https://github.com/dotnet/roslyn/issues/23222)

---
In reply to: [156566582](https://github.com/dotnet/roslyn/pull/23209#discussion_r156566582) [](ancestors = 156566582)